integrations/datadog/README.md

```diff
@@ -0,0 +1,114 @@
+# Materialize + Datadog
+
+<img width="1728" alt="Screenshot 2023-02-01 at 12 53 41" src="https://user-images.githubusercontent.com/11491779/216036715-9a4b4db7-8f93-4b6a-ac21-f7eb5a01d151.png">
+
+## Overview
+
+This is a demo of how to use Datadog to monitor Materialize using the OpenMetrics standard.
+
+Materialize collects information about its internal components in a [public catalog](https://materialize.com/docs/sql/system-catalog/mz_internal/), consumable by tools like [Prometheus SQL Exporter](https://github.com/justwatchcom/sql_exporter/), _"a service that runs user-defined SQL queries at flexible intervals and exports the resulting metrics via HTTP for Prometheus consumption"_.
+
+Datadog's OpenMetrics integration can consume from the Prometheus SQL Exporter endpoint and allows a way to monitor Materialize.
+
+<img width="1499" alt="Datadog" src="https://user-images.githubusercontent.com/11491779/216926716-cc0cdd46-7ea7-496f-8fa3-a4863fc1ce00.png">
+
+## Prerequisites
+
+- [Docker Compose](https://docs.docker.com/compose/install/)
+- [Materialize instance](https://materialize.com/cloud/)
+- [Datadog API Key](https://docs.datadoghq.com/account_management/api-app-keys/)
+
+## Running the demo
+
+1. Start by cloning the repository:
+
+    ```bash
+    git clone https://github.com/joacoc/DatadogOpenMetricsMaterialize.git
+    cd DatadogOpenMetricsMaterialize
+    ```
+
+2. Edit the `config.yaml` file and set your Materialize details under the connections:
+
+    ```yaml
+    connections:
+    - "postgres://<USER>:<PASSWORD>@<HOST>:<PORT>/materialize"
+    ```
+
+
+3. Edit the `docker-compose.yaml` file and set your Datadog API key:
+
+    ```yaml
+    environment:
+    - DD_API_KEY=${DD_API_KEY}
+    ```
+
+4. Start the demo:
+
+    ```bash
+    docker-compose up -d
+    ```
+
+5. Open your Datadog account and explore the `materialize.*` metrics.
+
+## Configuration overview
+
+The `config.yaml` file contains the configuration for the Prometheus SQL exporter. The `jobs` section divides into two:
+
+1. The `connections` section where you can configure the Prometheus SQL exporter to connect to multiple Materialize instances:
+
+```yaml
+connections:
+- "postgres://<USER>:<PASSWORD>@<HOST>:<PORT>/materialize"
```

Comment:
```suggestion
jobs:
- name: "global"
  interval: '1m'
  connections:
  - "postgres://MZ_USER:MZ_PASSWORD@MZ_HOST:6875/materialize"
```

Reply:
The way these configuration options is laid out can be improved in both demos. `connections` is a configuration option, just as `interval` and `name` are. Using "section" isn't that accurate. I can propose something after this is merged and we give it another pass to make the READMEs more consistent.